pre-docs/bolt_installing_modules.md

```diff
@@ -1,6 +1,40 @@
 # Installing modules
 
-Set up Bolt to download and install modules.
+Bolt is packaged with a collection of useful modules intended to support common work flows. The included modules are described in the [Puppetfile](https://github.com/puppetlabs/bolt/blob/master/Puppetfile) included in this repository. Modules and supporting documentation are publicly available on the [Puppet Forge](https://forge.puppet.com/).
+
+**NOTE**: Bundled Forge modules are NOT PROVIDED when bolt is installed via a ruby gem. It is recommended that bolt is [installed](https://github.com/puppetlabs/bolt/blob/master/pre-docs/bolt_installing.md) as a package for most use cases.
```

Comment:
@katelopresti Should this link to official docs, or does that translate to EasyDITA already?

Reply:
Using internal links between markdown files is fine. [Installed](bolt_installing.md)

Reply:
@katelopresti, does that mean that I need to refactor to _not_ use full `URLs` and instead use internal links? If so, does that include linking to another file in the github project (for example I want to link to the Puppetfile so the ref would look like `[Puppetfile](../Puppetfile)`?

Reply:
@donoghuc Don't worry about refactoring. Full URLs are fine, just not necessary for files within the pre-docs directory. For links outside of pre-docs, like Puppetfile, a full URL is what is used in the docs published to puppet.com